src/views/ListingView.vue

```diff
@@ -0,0 +1,103 @@
+<template>
```

Comment:
Original idea was to put this view in `IndividualView`, but this also works.

Remove the other file please